release-tools/build.make

```diff
@@ -101,9 +106,42 @@ push-%: container-%
 		fi; \
 	done
 
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+
+push-multiarch-%:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	make BUILD_PLATFORMS="windows amd64 .exe"
+	gcloud auth configure-docker
+	docker buildx create --use --name multiarchimage-buildertest
+	set -ex; \
+	pushMultiArch () { \
+                tag=$$1  ;\
+                docker buildx build --push -t $(MULTIARCH_IMAGE_NAME):amd64-linux-$$tag --platform=linux/amd64 -f Dockerfile.multiarch . ;\
```

Comment:
We loose the ability here to build different images in the same repo with different Docker files. Can you add something similar to `build-%` where it locates the Docker file with `$(shell if [ -e ./cmd/$*/Dockerfile ]; then echo ./cmd/$*/Dockerfile; else echo Dockerfile; fi)`?

Reply:
Updated `docker buildx build` command using approach of locating file.